src/check_pack.c

```diff
@@ -33,13 +34,20 @@
 #include "check_impl.h"
 #include "check_pack.h"
 
+#ifdef HAVE_WINDOWS_H
+#include <windows.h>
```

Comment:
What does windows.h provide for this file?

Reply:
Ah, this was an artifact from the previous patch. Removed, thanks.